docs/marker.md

```diff
@@ -17,6 +18,7 @@
 | `identifier` | `String` |  | An identifier used to reference this marker at a later date.
 | `rotation` | `Float` |  | A float number indicating marker's rotation angle, in degrees.
 | `draggable` | `<null>` |  | This is a non-value based prop. Adding this allows the marker to be draggable (re-positioned).
+| `tracksViewChanges` | `Boolean` |  | Controls whether the icon for this marker should be redrawn every frame.
```

Comment:
@PhamBaTho Is this prop iOS only?

Reply:
Only applicable for iOS. Android don't have issues with that.